packages/cozy-urls/src/helpers.js

```diff
@@ -0,0 +1,80 @@
+/* global __DEVELOPMENT__ */
+
+/* eslint-disable no-console */
+
+const VALID_PROTOCOLS = ['http:', 'https:']
+
+let APP_NAME = null
+
+const isDevelopment = () =>
+  typeof __DEVELOPMENT__ !== 'undefined' && !!__DEVELOPMENT__
+
+// Optional
+// Allow to specificy which application or library is throwing the error message
+export const setAppName = name => {
+  APP_NAME = name
+}
+export const getAppName = () => APP_NAME || 'The app'
+
+export const getSSL = (cozyURL, ssl) => {
+  if (typeof ssl !== 'undefined' && ssl !== null) return ssl
```

Comment:
We can only test `typeof ssl !== 'undefined'`, `null` being considered as a falsy value here.

Reply:
Ok I see. Let's do like this so 👍